packages/card-support/src/components/DownloadClientData/tests/utils.test.tsx

```diff
@@ -285,7 +300,7 @@ describe('components/DownloadClientData/utils/submitForm', () => {
       OpenSRPService,
       fixtures.locations,
       setSubmittingMock
-    );
+    ).catch(() => {});
```

Comment:
add a jest mock function and test if its called

Reply:
0318a30e715fd13ea4f55e95fa8b5c9269a73efa